### ISSUE TITLE
_sendJsonRpcRequest error due to requestId assigned a bool type

### ISSUE THREAD

**Issue by bobwith2bees** [WalletConnect/WalletConnectFlutterV2#74]
walletconnect_flutter_v2: ^2.0.1

In relay_client.dart _sendJsonRpcRequest, my app appears to get a bool value from the jsonRPC.sendRequest which is throwing a onRelayClientError error later.  I added the TypError to get the stack trace.  

relay_client.dart
```dart
    try {
      requestId = await jsonRPC.sendRequest(
        method,
        parameters,
        id,
      );
    } on StateError catch (_) {
      // Reconnect to the websocket
      // print('StateError, reconnecting: $_');
      await connect();
      requestId = await jsonRPC.sendRequest(
        method,
        parameters,
        id,
      );
    } on TypeError catch (e,s) {
      print ('_sendJsonRpcRequest - $e $s');
      rethrow;
    }
```

I see it fire when pairing
```
I/flutter (20219): 13:47:51 🐛 HomePage        - AppLifecycleState: AppLifecycleState.inactive.
I/flutter (20219): _sendJsonRpcRequest - type 'bool' is not a subtype of type 'String?' #0      RelayClient._sendJsonRpcRequest (package:walletconnect_flutter_v2/apis/core/relay_client/relay_client.dart:337:7)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #1      RelayClient.publish (package:walletconnect_flutter_v2/apis/core/relay_client/relay_client.dart:119:15)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #2      Pairing.sendResult (package:walletconnect_flutter_v2/apis/core/pairing/pairing.dart:457:5)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #3      SignEngine._onSessionSettleRequest (package:walletconnect_flutter_v2/apis/sign_api/sign_engine.dart:988:7)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): 13:48:00 🐛 WalletConnctSvc - Core.onRelayClientError  - error: type 'bool' is not a subtype of type 'String?'
```

I see it firing on a pairingPing
```
I/flutter (20219): 13:49:21 🐛 WalletConnctSvc - pingPairing - topic: c6852a9b1061ea095919dec049e7c79f7c44ebcc32fe26dca6f9321e3e78e6c9
I/flutter (20219): _sendJsonRpcRequest - type 'bool' is not a subtype of type 'String?' #0      RelayClient._sendJsonRpcRequest (package:walletconnect_flutter_v2/apis/core/relay_client/relay_client.dart:337:7)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #1      RelayClient.publish (package:walletconnect_flutter_v2/apis/core/relay_client/relay_client.dart:119:15)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #2      Pairing.sendRequest (package:walletconnect_flutter_v2/apis/core/pairing/pairing.dart:409:5)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): #3      Pairing.ping (package:walletconnect_flutter_v2/apis/core/pairing/pairing.dart:309:22)
I/flutter (20219): <asynchronous suspension>
I/flutter (20219): 13:49:21 🐛 WalletConnctSvc - Core.onRelayClientError  - error: type 'bool' is not a subtype of type 'String?'
``

**Comment by bobwith2bees** [comment]
FYI:
I see the issue start around May 1  with https://github.com/WalletConnect/WalletConnectFlutterV2/commit/3a4ad92e01376f81b3e9b0bc9c6f01ec972c607a

I tried deleting the app (in case there was a stored session on the device that was incompatible with the changes) but no luck.

**Comment by Luzzotica** [comment]
This should be resolved with v2.0.2 being pushed out now.
If it isn't let me know, otherwise I'll close this issue in 2-3 days.

**Comment by bobwith2bees** [comment]
Confirmed the issue is no longer present and likely fixed by commit https://github.com/WalletConnect/WalletConnectFlutterV2/commit/e212b16ce9dd30c1f2aa63ea00489da0f0b5850d

Closing.
